src/Microsoft.Performance.SDK/Processing/ColumnRole.cs

```diff
@@ -4,55 +4,48 @@
 namespace Microsoft.Performance.SDK.Processing
 {
     /// <summary>
-    ///     Defines the role of a column in a graph.
+    ///     Defines known string values of column roles.
```

Comment:
"Known" is a bit vague. Whomst is it known by? 😆 

What would you think of something like

Defines a common set of column roles that that are guaranteed to be successfully saved/restored in a serialized `<see cref="TableConfiguration" />`. Plugins may use any string value defined in this class to communicate column roles to any SDK driver. If a `<see cref="TableConfiguration" />` includes a column role value that is _not_ defined here, it is not guaranteed to 
1. Be de/serialized correctly
2. Be understood by an SDK driver